src/tasks/familiarweight.ts

```diff
@@ -116,6 +116,8 @@ export const FamiliarWeightQuest: Quest = {
             $familiars`Comma Chameleon, Homemade Robot`.every((fam) => have(fam)))
```

Comment:
We can just run the comma
```if ($familiars`Comma Chameleon, Homemade Robot`.every((fam) => have(fam))) && familiarWeight(chooseHeaviestFamiliar())) < 1 + 11 * get("homemadeRobotUpgrades")```

Reply:
Granted, that means that we might still use the comma (and a familiar jack) if the comma weight is higher but does not actually result in any turnsaves on `actualCost()`, but this is the same behaviour as the current PR. I'm amenable to changing that behaviour in the future